.github/workflows/ghsa-pr.yml

```diff
@@ -0,0 +1,260 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+
+name: "GHSA Pull Request Check"
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  ghsa_pr_checks:
+    if: >
+      github.event.pull_request.base.repo.fork == true &&
+      github.event.pull_request.base.repo.owner.login == 'tianocore' &&
+      startsWith(github.event.pull_request.base.repo.name, 'edk2-ghsa-')
+
+    name: GHSA Pull Request (CI) Checks
+    runs-on: ubuntu-24.04
+    container:
+      image: ghcr.io/tianocore/containers/fedora-40-test:c98ff99
+      credentials:
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    permissions:
+      contents: read
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - Package: "CryptoPkg"
+            ArchList: "IA32"
+          - Package: "CryptoPkg"
+            ArchList: "X64"
+          - Package: "CryptoPkg"
+            ArchList: "AARCH64"
+          - Package: "DynamicTablesPkg"
+            ArchList: "AARCH64"
+          - Package: "DynamicTablesPkg"
+            ArchList: "X64"
+          - Package: "FatPkg"
+            ArchList: "AARCH64"
+          - Package: "FatPkg"
+            ArchList: "X64"
+          - Package: "FmpDevicePkg"
+            ArchList: "AARCH64"
+          - Package: "FmpDevicePkg"
+            ArchList: "X64"
+          - Package: "IntelFsp2Pkg"
+            ArchList: "X64"
+          - Package: "IntelFsp2WrapperPkg"
+            ArchList: "X64"
+          - Package: "MdeModulePkg"
+            ArchList: "AARCH64"
+          - Package: "MdeModulePkg"
+            ArchList: "IA32"
+          - Package: "MdeModulePkg"
+            ArchList: "X64"
+          - Package: "MdePkg"
+            ArchList: "AARCH64"
+          - Package: "MdePkg"
+            ArchList: "IA32,X64"
+          - Package: "PcAtChipsetPkg"
+            ArchList: "X64"
+          - Package: "PrmPkg"
+            ArchList: "AARCH64"
+          - Package: "PrmPkg"
+            ArchList: "X64"
+          - Package: "SecurityPkg"
+            ArchList: "AARCH64"
+          - Package: "SecurityPkg"
+            ArchList: "X64"
+          - Package: "ShellPkg"
+            ArchList: "AARCH64"
+          - Package: "ShellPkg"
+            ArchList: "X64"
+          - Package: "SourceLevelDebugPkg"
+            ArchList: "X64"
+          - Package: "StandaloneMmPkg"
+            ArchList: "AARCH64"
+          - Package: "StandaloneMmPkg"
+            ArchList: "x64"
+          - Package: "UefiCpuPkg"
+            ArchList: "X64"
+          - Package: "UnitTestFrameworkPkg"
+            ArchList: "AARCH64"
+          - Package: "UnitTestFrameworkPkg"
+            ArchList: "X64"
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Set PATH
+      run: |
+        echo "${HOME}/.local/bin" >> $GITHUB_PATH
+        echo "new PATH=${PATH}"
+
+    - name: Configure Git Safe Directory

```

Comment:
Do we care to run stuart_pr_eval to skip any packages?

Reply:
For the original GHSA scenario it was not necessary. For the pivot to being a normal workflow, yes. There will also be some other changes like that made too.